docs/content/users/debugging-profiling/step-debugging.md

```diff
@@ -15,6 +15,7 @@ All IDEs basically work the same, listening on a port and reacting when they’r
 It will remain enabled until you start or restart the project.
 * Disable Xdebug for better performance when not debugging with `ddev xdebug off`.
 * `ddev xdebug status` will show Xdebug’s current status.
+* You may need to [open port 9003 in your firewall](https://stackoverflow.com/a/72591361) to allow Xdebug access. See also "Troubleshooting Xdebug" below.
```

Comment:
You could save someone a few seconds linking directly to that section:

```suggestion
* You may need to [open port 9003 in your firewall](https://stackoverflow.com/a/72591361) to allow Xdebug access. (See [Troubleshooting Xdebug](#troubleshooting-xdebug).)
```

Reply:
Thanks @mattstein, that's a great suggestion. Committed!